pkgs/servers/mastodon/default.nix

```diff
@@ -39,11 +40,11 @@ stdenv.mkDerivation rec {
 
   mastodonModules = stdenv.mkDerivation {
     pname = "${pname}-modules";
-    inherit src version;
+    inherit src version yarnHash;
 
     yarnOfflineCache = fetchYarnDeps {
       yarnLock = "${src}/yarn.lock";
-      sha256 = "sha256-e3rl/WuKXaUdeDEYvo1sSubuIwtBjkbguCYdAijwXOA=";
+      hash = yarnHash;
```

Comment:
I'd rather not add more sed logic to Nixpkgs, I think it's a good solution to just have it in a separate file, like the version and source.

Reply:
Wouldn't it then be a better idea to move it everything into one file?

Reply:
@SuperSandro2000 imho no, because different tools are generating these strings and it's much simpler to assume that the file can be completely overwritten